gitlab-docker/README.md

```diff
@@ -116,7 +116,7 @@ spack-mirror
    - `DOWNSTREAM_CI_REPO`:
      `http://root:<PASS>@gitlab:10080/root/spack-build.git`
    - `SPACK_RELEASE_ENVIRONMENT_REPO`:
-      `http://root:<PASS>@gitlab:10080/root/spack-env.git
+      `http://root:gitlab-secret<PASS>@gitlab:10080/root/spack-env.git`
```

Comment:
The instructions above mention how you need to replace `<PASS>` with your chosen password.  Can we leave this as-is?

Reply:
Whoops, you're right, I just goofed that up, but will fix it.  Sorry!